backend/src/main/java/com/redhat/cloud/notifications/auth/ConsoleIdentityProvider.java

```diff
@@ -179,4 +189,11 @@ private static ConsoleIdentity getRhIdentityFromString(String xRhIdHeader) {
         identity.rawIdentity = xRhIdHeader;
         return identity;
     }
+
+    private CurrentVertxRequest request() {
+        if (currentVertxRequest == null) {
+            currentVertxRequest = CDI.current().select(CurrentVertxRequest.class).get();
+        }
+        return currentVertxRequest;
+    }
```

Comment:
`CurrentVertxRequest` is a `@RequestScoped` bean while this code will store the value in an `@ApplicationScoped` bean.

Couldn't we inject `CurrentVertxRequest` instead of doing this?

Reply:
We can for sure. I probably did not know about the injectable bean. Or it wasn't yet when I wrote the previous code :) (But I guess it is the former reason)

Reply:
I replaced the old code with an injection of `RoutingContext` which is a `@RequestScoped` bean produced by `CurrentVertxRequest`.

Reply:
The tests logs show that `x-rh-rbac-org-id` is correctly set in the routing context for each query:
```
2022-09-21T09:03:26.7147864Z [INFO] Running com.redhat.cloud.notifications.routers.EndpointResourceTest
2022-09-21T09:03:26.7964647Z 2022-09-21 09:03:26,773 INFO  [info] (executor-thread-1) "127.0.0.1 - user 21/Sep/2022:09:03:26 +0000 "POST /api/integrations/v1.0/endpoints HTTP/1.1" 400 34 orgId=empty"
2022-09-21T09:03:26.7965934Z 2022-09-21 09:03:26,784 INFO  [info] (executor-thread-1) "127.0.0.1 - user 21/Sep/2022:09:03:26 +0000 "POST /api/integrations/v1.0/endpoints HTTP/1.1" 400 34 orgId=empty"
2022-09-21T09:03:26.9352391Z 2022-09-21 09:03:26,847 INFO  [info] (executor-thread-1) "127.0.0.1 - user 21/Sep/2022:09:03:26 +0000 "POST /api/integrations/v1.0/endpoints HTTP/1.1" 200 437 orgId=testWebhookAttributes2"
2022-09-21T09:03:26.9354220Z 2022-09-21 09:03:26,864 INFO  [info] (executor-thread-1) "127.0.0.1 - user 21/Sep/2022:09:03:26 +0000 "GET /api/integrations/v1.0/endpoints/3a8e330d-4362-429d-8900-25a6495d8f8e HTTP/1.1" 200 437 orgId=testWebhookAttributes2"
2022-09-21T09:03:27.2356295Z 2022-09-21 09:03:27,171 INFO  [info] (executor-thread-1) "127.0.0.1 - test-user 21/Sep/2022:09:03:27 +0000 "DELETE /api/integrations/v1.0/endpoints/email/subscription/rhel/tester/instant HTTP/1.1" 404 18 orgId=test-subscription-org-id"
```